staging/src/k8s.io/component-base/logs/api/v1/options.go

```diff
@@ -310,6 +310,33 @@ func AddGoFlags(c *LoggingConfiguration, fs *flag.FlagSet) {
 	addFlags(c, goFlagSet{FlagSet: fs})
 }
 
+// SetFlags applies the logging flags from the given flag set to the given
+// configuration. Fields for which the corresponding flag was not used are left
+// unmodified.
+func SetFlags(fs *pflag.FlagSet, c *LoggingConfiguration) error {
```

Comment:
do you want to make this part of the log api?

should not handle this directly in kube-proxy since it already has a precedent overriding a flag https://github.com/kubernetes/kubernetes/issues/119864#issuecomment-1671642653

This way you can come with a more universal solution in 1.28

Reply:
I think the final solution may be something like this. Doing this entirely in kube-proxy as a hot fix now is easier than trying to get it implemented properly here.

OTOH, this is just a Go API. It can be changed at any time without breaking the API of components using it, only compilation of the component may break...

Reply:
> Perhaps `CopyFlags(from *pflag.FlagSet, to *LoggingConfiguration)`?

OverrideLogsFromFlags(from *pflag.FlagSet, to *LoggingConfiguration) ?